env-proto.sh

```diff
@@ -17,9 +17,9 @@ go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.27.1
 
 # Substitute VERSION for the current released version.
 # Substitute BINARY_NAME for "buf", "protoc-gen-buf-breaking", or "protoc-gen-buf-lint".
-VERSION="0.51.1" && \
+VERSION="1.0.0-rc8" && \
 BINARY_NAME="buf" && \
-  curl -sSL \
+  sudo curl -sSL \
     "https://github.com/bufbuild/buf/releases/download/v${VERSION}/${BINARY_NAME}-$(uname -s)-$(uname -m)" \
     -o "${BIN_DIR}/${BINARY_NAME}" && \
-  chmod +x "${BIN_DIR}/${BINARY_NAME}"
\ No newline at end of file
+  sudo chmod +x "${BIN_DIR}/${BINARY_NAME}"
```

Comment:
Is there really no way of doing this without calling `sudo` from a build script? :flushed: :thinking:

Reply:
My bad, I think this was just something I had to do in my env, don't think this is universally necessary.

Reply:
Ok! :blush: Thanks!